src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md

```diff
@@ -319,7 +320,7 @@ Calling deploy() on a ContractFactory will start the deployment, and return a Pr
 
 We’re finally ready to deploy our smart contract! Navigate back to the root of your project directory, and in the command line run:
 
-    npx hardhat run scripts/deploy.js --network ropsten
+    npx hardhat run scripts/deploy.js
```

Comment:
I think we should be keeping a test network here for the deploy to protect user funds and avoid confusion.

Reply:
If you think there is a better testnet though, please do change

Reply:
I thought this was being ignored, as the contents of scripts/deploy.js in the same file don't look at command line arguments. I realize now this flag is meant for `hardhat`, so I'll add it back and place it before `run` to make that clear.